plugins/teststeps/sshcmd/sshcmd.go

```diff
@@ -176,8 +177,10 @@ func (ts *SSHCmd) Run(cancel, pause <-chan struct{}, ch test.TestStepChannels, p
 			}
 		}()
 		// run the remote command and catch stdout/stderr
-		var stdout, stderr bytes.Buffer
-		session.Stdout, session.Stderr = &stdout, &stderr
+		var stdoutBuf, stderrBuf bytes.Buffer
+		session.Stdout = io.MultiWriter(os.Stdout, &stdoutBuf)
+		session.Stderr = io.MultiWriter(os.Stderr, &stderrBuf)
```

Comment:
I think this is wrong to non-optionally hardcode `os.Stdout`/`os.Stderr` here. Can we add an option which enables this `os.Stdout`/`os.Stderr` injection? And add a comment why such options could be needed.